test/integration/bad/reduce_sum/bad_return_type_static.abc

```diff
@@ -1,5 +1,5 @@
 functions {
-  int my_func_0(int start, int end, real[] y_slice) {
+  int my_func_0(real[] y_slice, int start, int end) {
```

Comment:
This one is on me, but can you change the extension. Dont know why I put .abc :)